examples/src/components/GeoPushpinChartWithCategoryLegendExample.jsx

```diff
@@ -0,0 +1,53 @@
+// (C) 2020 GoodData Corporation
+import React, { Component } from "react";
+import { GeoPushpinChart, Model } from "@gooddata/react-components";
+
+import "@gooddata/react-components/styles/css/main.css";
+
+import { projectId, cityCoordinatesUri, populationUri, densityUri, stateNamesUri } from "../utils/fixtures";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const colorMeasure = Model.measure(densityUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Density");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const segmentByAttribute = Model.attribute(stateNamesUri).localIdentifier("segment");
+
+export class GeoPushpinChartWithCategoryLegendExample extends Component {
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartWithCategoryLegendExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartWithCategoryLegendExample onError", ...params);
+    }
+
+    render() {
+        const MAPBOX_TOKEN = process.env.EXAMPLE_MAPBOX_ACCESS_TOKEN;
+        const style = { height: "500px", position: "relative" };
+        return (
+            <div style={style} className="s-geo-pushpin-chart-category">
+                <GeoPushpinChart
+                    projectId={projectId}
+                    location={locationAttribute}
+                    size={sizeMeasure}
+                    color={colorMeasure}
+                    segmentBy={segmentByAttribute}
+                    onLoadingChanged={this.onLoadingChanged}
+                    onError={this.onError}
+                    config={{
+                        mapboxToken: MAPBOX_TOKEN,
+                    }}
```

Comment:
We should arrange React props as following
```
                    projectId={projectId}
                    location={locationAttribute}
                    size={sizeMeasure}
                    color={colorMeasure}
                    segmentBy={segmentByAttribute}
                    config={{
                        mapboxToken: MAPBOX_TOKEN,
                    }}
                    onZoomChanged={...}
                    onCenterPositionChanged={...}
                    onLoadingChanged={this.onLoadingChanged}
                    onError={this.onError}
```

Besides, we should add `onZoomChanged` and `onCenterPositionChanged` to guide user better
Please apply to all geo chart

Reply:
fixed